packages/sanity/src/core/form/inputs/DateInputs/DateTimeInput.tsx

```diff
@@ -90,28 +106,46 @@ export function DateTimeInput(props: DateTimeInputProps) {
   )
 
   const formatInputValue = useCallback(
-    (date: Date) => format(date, `${dateFormat} ${timeFormat}`),
-    [dateFormat, timeFormat],
+    (date: Date) => format(date, `${dateFormat} ${timeFormat}`, {timezone: displayTimezone}),
+    [dateFormat, timeFormat, displayTimezone],
   )
 
   const parseInputValue = useCallback(
-    (inputValue: string) => parse(inputValue, `${dateFormat} ${timeFormat}`),
-    [dateFormat, timeFormat],
+    (inputValue: string): ParseResult =>
+      parse(inputValue, `${dateFormat} ${timeFormat}`, displayTimezone),
+    [dateFormat, timeFormat, displayTimezone],
   )
   const calendarLabels: CalendarLabels = useMemo(() => getCalendarLabels(t), [t])
   return (
-    <CommonDateTimeInput
-      {...elementProps}
-      calendarLabels={calendarLabels}
-      onChange={handleChange}
-      deserialize={deserialize}
-      formatInputValue={formatInputValue}
-      parseInputValue={parseInputValue}
-      placeholder={schemaType.placeholder}
-      selectTime
-      serialize={serialize}
-      timeStep={timeStep}
-      value={value}
-    />
+    <Tooltip
+      placement="bottom-start"
+      content={
+        displayTimezone ? (
+          <Inline space={1}>
+            <EarthGlobeIcon />
+            <Text size={1} muted>
+              {t('inputs.datetime.timezone-information-text', {timezone: displayTimezone})}
+            </Text>
+          </Inline>
+        ) : (
+          ''
+        )
```

Comment:
Would it be better to not wrap the input in the tooltip if there's no display timezone? Minor, but seems like it might add unnecessary calculation of content/tooltip location etc.